org.eclipse.winery.generators.ia/src/test/java/org/eclipse/winery/generators/ia/GeneratorTest.java

```diff
@@ -13,126 +13,96 @@
-        System.out.println(generateProject);
+    public void testMultipleOpsWithInOutParams() throws Exception {
+        TInterface i = new TInterface();
+        i.setName("http://www.example.org/interfaces/lifecycle");
+
+        TOperation op;
+        TOperation.InputParameters input;
+        TOperation.OutputParameters output;
+        TParameter param;
+
+        op = new TOperation();
+        op.setName("install");
+        i.getOperation().add(op);
+
+        input = new TOperation.InputParameters();
+        param = new TParameter();
+        param.setName("VMIP");
+        param.setType("xs:string");
+        input.getInputParameter().add(param);
+        param = new TParameter();
+        param.setName("DBMSUsername");
+        param.setType("xs:string");
+        input.getInputParameter().add(param);
+        op.setInputParameters(input);
+
+        output = new TOperation.OutputParameters();
+        param = new TParameter();
+        param.setName("Output");
+        param.setType("xs:string");
+        output.getOutputParameter().add(param);
+        op.setOutputParameters(output);
+
+        op = new TOperation();
+        op.setName("uninstall");
+        i.getOperation().add(op);
+
+        input = new TOperation.InputParameters();
+        param = new TParameter();
+        param.setName("SomeLongParameterName");
+        param.setType("xs:string");
+        input.getInputParameter().add(param);
+        param = new TParameter();
+        param.setName("Port");
+        param.setType("xs:string");
+        input.getInputParameter().add(param);
+        op.setInputParameters(input);
+
+        op.setOutputParameters(output);
+
+        Generator gen = new Generator(i, "org.opentosca.ia.test", new URL("http://test.com"), "TestMultipleOpsWithInOutParams", wd.toFile());
+        gen.generateProject();
     }
 
     @Test
-    public void testNoParams() throws Exception {
-        TInterface tinterface = new TInterface();
-        tinterface.setName("TestNoParams");
-
-        TOperation opIn = new TOperation();
-        opIn.setName("OpNoParams");
-        tinterface.getOperation().add(opIn);
+    public void testOneOpNoParams() throws Exception {
+        TInterface i = new TInterface();
+        i.setName("http://www.example.org/interfaces/lifecycle");
 
-        TNodeType nodeType = new TNodeType();
-        nodeType.setName("test");
-        nodeType.setTargetNamespace("http://asd.com");
+        TOperation op = new TOperation();
+        op.setName("install");
+        i.getOperation().add(op);
 
-        Generator gen = new Generator(tinterface, "org.opentosca.ia", new URL("http://asd.com"), "testname", WORKING_DIR.toFile());
-        Path generateProject = gen.generateProject();
-        System.out.println(generateProject);
+        Generator gen = new Generator(i, "org.opentosca.ia.test", new URL("http://test.com"), "TestOneOpNoParams", wd.toFile());
+        gen.generateProject();
```

Comment:
Why don't you compare your expected file against the generated one?

Reply:
See above.